core/src/main/scala/org/apache/spark/rdd/PipedRDD.scala

```diff
@@ -205,6 +206,8 @@ private[spark] class PipedRDD[T: ClassTag](
       private def propagateChildException(): Unit = {
         val t = childThreadException.get()
         if (t != null) {
+          logError(s"Caught exception ${t.getMessage} while running pipe(). Command ran: " +
```

Comment:
the exception message might be very long. would be better to put this at the end rather than in the middle.